test/test-mongocrypt-endpoint.c

```diff
@@ -40,8 +42,8 @@ _test_mongocrypt_endpoint (_mongocrypt_tester_t *tester)
    BSON_ASSERT (mongocrypt_status_ok (status));
    _mongocrypt_endpoint_destroy (endpoint);
 
-   endpoint =
-      _mongocrypt_endpoint_new ("kevin.keyvault.azure.net:443", -1, status);
+   endpoint = _mongocrypt_endpoint_new (
+      "kevin.keyvault.azure.net:443", -1, NULL /* opts */, status);
```

Comment:
Did we want that hostname in there non-scrubbed?

Reply:
I checked, and I do still have a test key vault in Azure from long ago. But it is under a different endpoint. I am OK leaving this as `kevin.keyvault.azure.net`.